siliconcompiler/tools/vpr/vpr.py

```diff
@@ -98,10 +98,15 @@ def runtime_options(chip, tool='vpr'):
 
     # Routing graph XML:
     rr_graphs = chip.find_files('tool', 'vpr', 'task', task, 'file', 'rr_graph',
+                                missing_ok=True,
                                 step=step, index=index)
```

Comment:
I think to be consistent with everything else, you should
```
if chip.valid('tool', 'vpr', 'task', task, 'file', 'rr_graph') and  chip.get('tool', 'vpr', 'task', task, 'file', 'rr_graph',
                                step=step, index=index):
  rr_graph =  chip.find_files('tool', 'vpr', 'task', task, 'file', 'rr_graph',
                                step=step, index=index)
else:
  rr_graph = []
```

Reply:
This ripples and breaks the check I have for not passing an rr graph to VPR successfully; will need to brainstorm options when we discuss other topics.